SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -24,155 +23,6 @@ import WebKit
 
 let SplunkRumVersionString = "0.8.0"
 
-/**
- Default maximum size of the disk cache in bytes.
- */
-public let DEFAULT_DISK_CACHE_MAX_SIZE_BYTES: Int64 = 25 * 1024 * 1024
-
-/**
- Optional configuration for SplunkRum.initialize()
- */
-@objc public class SplunkRumOptions: NSObject {
```

Comment:
Is the removal from this file transparent to end users? I don't know enough here, but say someone is importing `SplunkRumOptions`, is there a chance the moving of this class will break someone's code?